examples/README.md

```diff
@@ -516,6 +516,222 @@ tiki-taka v2 algorithm that achieves better performance than SGD.
 Here it is illustrated how the analog tiles can be directly used to
 implement an analog mat-vec (without a pytorch layer).
 
+## Example 24: [`24_bert_on_squad.py`]
+
+This example is adapted from
+https://github.com/huggingface/notebooks/blob/main/examples/question_answering.ipynb
+
+The example loads a pre-trained BERT model trained on
+the SQuAD dataset. It then applies `convert_to_analog()`
+to examine the effects of `drift_analog_weights()` on inference performance at
+different weight noise levels. Tensorboard is used to display the SQuAD
+metrics evaluated using the model at various times after training completed.
+
+Commandline arguments can be used to control certain options
+```python
+PARSER.add_argument("-d", "--digital",
+                    help="Add to use digital inference",
+                    action="store_true")
+PARSER.add_argument("-i", "--ideal",
+                    help="Add to use ideal config instead of default noisy one",
+                    action="store_true")
+PARSER.add_argument("-w", "--wandb",
+                    help="Add to use wandb",
+                    action="store_true")
+PARSER.add_argument("-n", "--noise",
+                    help="Weight noise",
+                    default=0.0175,
+                    type=float)
+```
+
+### AiMOS NPL Environment Setup
+
+Use `ssh` to connect to one of the [landing pads](https://docs.cci.rpi.edu/landingpads/)
+From there, connect to the NPL cluster with `ssh nplfen01`.
+
+Setup a proxy with
+```bash
+export http_proxy=http://proxy:8888
+export https_proxy=$http_proxy
+```
+Adding these lines to the `~/.bashrc` will ensure these are called upon every login.
+
+Follow the [installation instructions](https://docs.cci.rpi.edu/software/Conda/) to install conda.
+Then, create an environment with `conda create --name <env-name>`,
+and run `conda activate <env-name>`.
+
+After activation, install the necessary
+[dependences](https://aihwkit.readthedocs.io/en/latest/advanced_install.html),
+and run the following commands:
+
+`conda install pytorch torchvision torchaudio pytorch-cuda=11.6 -c pytorch -c nvidia`
```

Comment:
can we add these dependencoes to the setup.py so that one can just use something like  
`pip install aihwkit[bert]`
to install all dependencies as we did for the visualization etc. ?